homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -150,6 +150,12 @@ def convert_and_round(
         unit_type=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "climate_status": BMWSensorEntityDescription(
```

Comment:
What are the possible states of this sensor?

Reply:
The possible values are available in [this enum](https://github.com/bimmerconnected/bimmer_connected/blob/master/bimmer_connected/vehicle/climate.py#L11-L18).